src/Service/KeeperCommon.h

```diff
@@ -37,9 +37,12 @@ struct RequestForSession
     int64_t session_id;
     Coordination::ZooKeeperRequestPtr request;
 
-    /// measured in millisecond
+    /// When request create, millisecond count since system start up, used for calculate request duration
     int64_t create_time{};
 
+    /// When leader append write request, millisecond count since 1970-1-1, used for node stat ctime and mtime.
+    int64_t process_time{};
```

Comment:
Why we need the `process_time `?

Reply:
std::chrono::steady_clock is a relative clock used for measuring time intervals. We cannot use it for a node's ctime and mtime. We need an additional std::chrono::system_clock to store the modification and creation times of a node.

Reply:
Set`create_time` to system clock?

Reply:
> Set create_time to system clock?

`create_time` is when node see this user request and create it as session request
`process_time` is when RequestAccumulator sees this session request and tries to commit it. 
We assign the leader's system_clock as `process_time` to every request before `raft_instance->append_entries()` to ensure the sequential order of `process_time`, meaning the `process_tim`e of an earlier processed request should be less than that of a later one. This ensures the relative accuracy of a node's `ctime` and `mtime`. Due to the pipeline and forwarding processes, there might be cases where the `create_time` of an earlier processed request is greater than a later one, causing confusion in the node's `ctime` and `mtime`. `create_time` and `process_time` should be distinct: one is relative, used to measure request duration, and the other is absolute, used to record the historical operation time of the node."

Reply:
@JackyWoo Please review when you have time.

Reply:
Sure, but there might be some delay.